examples/upbound-aws-provider/composite-resources/database-examples/aurora/README.md

```diff
@@ -0,0 +1,240 @@
+# Steps to to deploy aurora rds cluster
+This example deploys an aurora postgresql database cluster when applied the claim. Below are the steps to execute to
+create a cluster and connect the cluster from a pod with an psql client.
+
+## Pre-requisites
+ - [Upbound AWS Provider Crossplane Blueprint Examples](../../README.md)
+
+
+### create 2 role, one for RDS Monitoring and one for RDS Proxy
+
+Create an IAM role and attach policy to the role (This role is required for aurora to perform monitoring)
```

Comment:
Why does this need to be done outside of Crossplane?

Reply:
Hi @nabuskey , I talked to @biswajitbabu and for the first pass we can have it in the Readme and then he will do another iteration to add it into the composition.

Reply:
I see. In that case, let's at least put a disclaimer or something in readme and create an issue to track it. I suspect the first thing users are going to wonder when reading this is "why do we have to create roles manually?". Having an issue to track this will be great to invite others to fix this if it doesn't get done in the near future.

Reply:
The changes are done as discussed.